lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,28 @@
-import React from 'react';
+import React from "react";
+import PropTypes from "prop-types";
 
 const Card = props => {
   return (
     <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
+      <div className="headline">{props.card.headline}</div>
       <div className="author">
         <div className="img-container">
-          <img src={'' /* image source goes here */} />
+          <img src={props.card.img} alt={props.card.headline} />
```

Comment:
Good work on this part. I love that you included an alt tag. That's great practice for accessibility!